code/modules/antagonists/blob/structures/_blob.dm

```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/mob/blob.dmi'
 	light_range = 2
 	desc = "A thick wall of writhing tendrils."
-	density = FALSE //this being false causes two bugs, being able to attack blob tiles behind other blobs and being unable to move on blob tiles in no gravity, but turning it to 1 causes the blob mobs to be unable to path through blobs, which is probably worse.
+	density = TRUE
```

Comment:
Did you manage to figure out how to get blob spores to path through blobs with this change?

Reply:
@TemporalOroboros canastarpass checks pass flags  it should work

Reply:
> @TemporalOroboros canastarpass checks pass flags it should work

this completely stops blob spores from pathing btw